Add SSO to frontend (#114)
bcgov/GDX-Analytics-URL-Shortener: backend/README.md

 
 **`Step 4`**. In the /backend folder, copy '.env.template' to a new file '.env'. This .env file is used to store the BC Government Single Sign-on (SSO) secret and to manage other settings.  All explanation/uses for env variables are given in comments in the .env.template file.
  

>   Check more details at https://bcgov.github.io/sso-requests; click Login; select the Request ID - 00005780	Project Name - URL-SHORTENER-FRONTEND;

Reviewer: ```suggestion
To verify your SSO_CLIENT_ID, visit https://bcgov.github.io/sso-requests, log in with your IDIR, and select the relevant Frontend project. You can download the integration details as a JSON file.
```

  
  Note that .gitignore lists .env - it will not be added to the repository. 
 
 **`Step 5`**. From the /backend folder, run `npm install` in the terminal to install required dependencies.
 
 **`Step 6`**. From the /backend folder, run `npm run start` in the terminal to start the backend app.

 **`Step 7`**. Complete the the Frontend installation steps in the /frontend folder README.

## Details

## Single Sign-on Pre-requisite

> - We needed a BC Government Single Sign-on integration with client type `public` before we could start using SSO in the URL-S app

Reviewer: ```suggestion
- A BC Government Single Sign-on integration with a client type of `public` is required to use SSO in the URL-S app.
```
